### ISSUE TITLE
ImportError: bad magic number in 'yolo3': b'\x03\xf3\r\n'

### ISSUE THREAD

**Issue by nnzs** [yehengchen/Object-Detection-and-Tracking#14]
Why is this?

**Comment by yehengchen** [comment]
Please go to your home directory. And then:

`sudo find . -name "*.pyc" -exec rm -f {} `

**Comment by nnzs** [comment]
> @nnzs Please go to your home directory. And then:
> 
> `sudo find . -name "*.pyc" -exec rm -f {} `

thanks a lot

**Comment by nnzs** [comment]
> sudo find . -name "*.pyc" -exec rm -f {}
Do I need to prepare my own test video?

**Comment by nnzs** [comment]
> @nnzs Please go to your home directory. And then:
> 
> `sudo find . -name "*.pyc" -exec rm -f {} `

UnboundLocalError: local variable 'track' referenced before assignment

**Comment by yehengchen** [comment]
Yep, Here is testing video in Google Drive. [ [TownCentreXVID.avi] ](https://drive.google.com/file/d/14jo0UIfkCt2PdsjW4cC0tHM31TkiWbsf/view)

**Comment by nnzs** [comment]
> @nnzs Yep, Here is testing video in Google Drive. [ [TownCentreXVID.avi] ](https://drive.google.com/file/d/14jo0UIfkCt2PdsjW4cC0tHM31TkiWbsf/view)

Thank you so much for your timely reply！！

**Comment by huycong2798** [comment]
> @nnzs Yep, Here is testing video in Google Drive. [ [TownCentreXVID.avi] ](https://drive.google.com/file/d/14jo0UIfkCt2PdsjW4cC0tHM31TkiWbsf/view)

Hello there, can i get the testing video of this one please. 
![image](https://user-images.githubusercontent.com/45420711/64502085-6892a080-d2ee-11e9-880c-4c653641e75b.png)

**Comment by nnzs** [comment]
When I test with a camera, I can't detect person.Why？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Bobby Chen"<notifications@github.com&gt;;
发送时间:&nbsp;2019年9月7日(星期六) 晚上8:33
收件人:&nbsp;"yehengchen/Object-Detection-and-Tracking"<Object-Detection-and-Tracking@noreply.github.com&gt;;
抄送:&nbsp;"嗯."<491696554@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [yehengchen/Object-Detection-and-Tracking] ImportError: bad magic number in 'yolo3': b'\x03\xf3\r\n' (#14)




Yep, Here is testing video in Google Drive.  [TownCentreXVID.avi]

**Comment by yehengchen** [comment]
@nnzs  I think maybe you should train the weights for your custom data.

**Comment by cmendozab** [comment]
> @nnzs Yep, Here is testing video in Google Drive. [ [TownCentreXVID.avi]](https://drive.google.com/file/d/14jo0UIfkCt2PdsjW4cC0tHM31TkiWbsf/view)

Hello yehengchen, 
Do you have the "TownCentre-output-HOGPedestrians.top" file? It should come with the "TownCentreXVID.avi" file.
